website/content/docs/connect/observability/access-logs.mdx

```diff
@@ -0,0 +1,255 @@
+You can use these fields to customize the access logs that proxies emit. 
+
+Custom logs can be either JSON format or text format.
+
+### JSON format
+
+You can format access logs in JSON so that you can parse them with Application Monitoring Platforms (APMs).
+
+To use a custom access log, in the `proxy-defaults` configuration entry, set [`JSONFormat`](/docs/connect/config-entries/proxy-defaults#jsonformat) to the string representation of the desired JSON.
+
+Nesting is supported.
+
+<CodeTabs tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+
+```hcl
+Kind      = "proxy-defaults"
+Name      = "global"
+AccessLogs {
+  Enabled = true
+  JSONFormat = <<EOF
+{
+    "myCustomKey" : {
+        "myStartTime" : "%START_TIME%",
+        "myProtocol" : "%PROTOCOL%"
+    }
+}
+EOF
+}
+```
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ProxyDefaults
+metadata:
+  name: global
+spec:
+  accessLogs:
+    enabled: true
+    jsonFormat: |
+      {
+        "myCustomKey" : {
+          "myStartTime" : "%START_TIME%",
+          "myProtocol" : "%PROTOCOL%"
+        }
+      }
+```
+
+```json
+{
+  "Kind": "proxy-defaults",
+  "Name": "global",
+  "AccessLogs": {
+    "Enabled": true,
+    "JSONFormat": "{ \"myCustomKey\" : { \"myStartTime\" : \"%START_TIME%\", \"myProtocol\" : \"%PROTOCOL%\"} }"
+  }
+}
+```
+
+</CodeTabs>
+
+### Text format
+
+To use a custom access log formatted in plaintext, in the `proxy-defaults` configuration entry, set [`TextFormat`](/docs/connect/config-entries/proxy-defaults#textformat) to the desired customized string.
+
+New lines are automatically added to the end of the log to keep each access log on its own line in the output.
+
+<CodeTabs tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+
+```hcl
+Kind      = "proxy-defaults"
+Name      = "global"
+AccessLogs {
+  Enabled = true
+  TextFormat = "MY START TIME: %START_TIME%, THIS CONNECTIONS PROTOCOL IS %PROTOCOL%" 
+}
+```
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ProxyDefaults
+metadata:
+  name: global
+spec:
+  accessLogs:
+    enabled: true
+    textFormat: "MY START TIME: %START_TIME%, THIS CONNECTIONS PROTOCOL IS %PROTOCOL%" 
+```
+
+```json
+{
+  "Kind": "proxy-defaults",
+  "Name": "global",
+  "AccessLogs": {
+    "Enabled": true,
+    "JSONFormat": "MY START TIME: %START_TIME%, THIS CONNECTIONS PROTOCOL IS %PROTOCOL%" 
+  }
+}
+```
+
+</CodeTabs>
+
+
+## Kubernetes
+
+As part of its normal operation, the Envoy debugging logs for the `consul-dataplane`, `envoy`, or `envoy-sidecar` containers are written to `stderr`.
+The access log [`Type`](/docs/connect/config-entries/proxy-defaults#type) is set to `stdout` by default for access logs when enabled.
+Use a log aggregating solution to separate the machine-readable access logs from the Envoy process debug logs.
+
+## Write to a file
+
+You can write access logs to a file on the host where Envoy runs.
+The is most relevant when running Envoy on a VM.
```

Comment:
I was thinking from a perspective that it would be subjectively bad to do this for a containerized deployed such as consul-k8s. I think it's fairly obvious, and I don't expect many people to try it, so I think it's fair to leave out the info.